### PR TITLE
feat: added code snippet

### PR DESCRIPTION
**Pull request by AshutoshM29**
Added code snippet for a few components in the library!
List Preview: https://deploy-preview-11--uiclean.netlify.app/pages/list/list
Navigation Preview: https://deploy-preview-11--uiclean.netlify.app/pages/navigation/navigation
Modal Preview: https://deploy-preview-11--uiclean.netlify.app/pages/modal/modal

Please review and provide feedback

### CODE REVIEW

File: Pages/List/list.html [hunk 1]
```diff
@@ -66,6 +66,14 @@ <h3>Numbered List - I</h3>
                 <li>list Item3</li>
             </ol>
         </div>
+        <p class="body-r">Copy the given code to create Numbered List - I</p>
+
+        <iframe
+  src="https://carbon.now.sh/embed?bg=rgba%283%2C26%2C77%2C1%29&t=blackboard&wt=none&l=htmlmixed&width=680&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Fira+Code&fs=13px&lh=152%25&si=false&es=2x&wm=false&code=%253Col%253E%250A%2509%253Cli%253Elist%2520Item1%253C%252Fli%253E%250A%2509%253Cli%253Elist%2520Item2%253C%252Fli%253E%250A%2509%253Cli%253Elist%2520Item3%253C%252Fli%253E%250A%253C%252Fol%253E"
+  style="width: 50rem; height: 40rem; border:0; transform: scale(1); overflow:hidden;"
```
Comment (kaustubh-jsr) [inline Pages/List/list.html:73]: You can remove the inline styles, and may be give a class and extract this out, for code iframe tags.

**Review by kaustubh-jsr — COMMENTED** [review]
You can check my comment. Also if it was not intended, then please check the side navigation bar, which doesn't start from the top.

File: Pages/Navigation/navigation.html [hunk 2]
```diff
@@ -51,69 +51,74 @@ <h3>Components</h3>
...
+                    </li>
+                    <li class="list-inline-item">
+                        <a href="/">Item2</a>
+                    </li>
+                    <li class="list-inline-item">
+                        <a href="/">Item3</a>
+                    </li>
+                </ul>
+            </div>
+            <label class="search-bar">
```
Comment (gauravsinhaweb) [inline Pages/Navigation/navigation.html:75]: use explicit labelling. Apply it for all.

**Review by gauravsinhaweb — APPROVED** [review]
Good to merge after changes.

File: Pages/List/list.html [hunk 3]
```diff
@@ -152,6 +208,12 @@ <h3>Default List w/ No Bullets</h3>
                     <li>Item3</li>
                 </ul>
             </div>
+            <p class="body-r">Copy the given code to create Default List with No Bullets</p>
+            <iframe
+  src="https://carbon.now.sh/embed?bg=rgba%283%2C26%2C77%2C1%29&t=blackboard&wt=none&l=htmlmixed&width=680&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Fira+Code&fs=13px&lh=152%25&si=false&es=2x&wm=false&code=%253Cul%2520class%253D%2522list-style-none%2522%253E%250A%2509%253Cli%253EItem1%253C%252Fli%253E%250A%2509%253Cli%253EItem2%253C%252Fli%253E%250A%2509%253Cli%253EItem3%253C%252Fli%253E%250A%253C%252Ful%253E"
+  style="width: 50rem; height: 40rem; border:0; transform: scale(1); overflow:hidden;"
+  sandbox="allow-scripts allow-same-origin">
```
Comment (rahulk31) [inline Pages/List/list.html:215]: I am not sure about these inline styles for iframes if they comes along with carbon. If no then may be you can plan to use a class for these properties as they are repeating.

**Review by rahulk31 — APPROVED** [review]
Overall code structure looks great and goos use of classes. Can be merged 👍

File: Pages/List/list.html (hunk 3 above)
Comment (AshutoshM29) [reply Pages/List/list.html:215]: Whenever I import iframe from carbon it comes with this inline style, the only thing that I have changed is the unit of width and height (from px to rem)

**Pull request merged**